llvm/unittests/IR/MetadataTest.cpp

```diff
@@ -2477,9 +2477,12 @@ typedef MetadataTest TicketNodeTest;
 
 TEST_F(TicketNodeTest, get) {
   ticketmd::DigestType Digest = getDigest();
-  TicketNode *D = TicketNode::get(Context, "foo", Digest,
-                                  GlobalValue::ExternalLinkage, false);
-  EXPECT_EQ(GlobalValue::ExternalLinkage, D->getLinkage());
+  TicketNode *D = TicketNode::get(
+      Context, "foo", Digest, GlobalValue::LinkageTypes::ExternalLinkage,
+      GlobalValue::VisibilityTypes::DefaultVisibility, false);
```

Comment:
It looks as though *all* of the tests (system and unit) use default visibility. If true, we should probably add tests for the other values (partly because we‘ve got that bit-twiddling in the implementation).

Reply:
The visibility tests are added.